### ISSUE TITLE
Doesn't work with string literals in triple single quotes

### ISSUE THREAD

**Issue by papandreou** [fibo/SQL-tokenizer#1]
```
require('sql-tokenizer')()(`SELECT '''foo''' from bar`)
[ 'SELECT', ' ', "'''foo'", "''" ]
```

Expected:

```
[ 'SELECT', ' ', "'''foo'''", ' ', 'from', 'bar' ]
```

**Comment by fibo** [comment]
thank you for reporting it

I have few questions:

1. Just my curiosity, I have a test that checks for escaped quotes like `select 'O''Reilly' from mytable`, I do not remember use cases for three quotes, but I know there are some. Can you please tell me the use case, if I am not wrong it is used when you generate SQL from a SQL query.

2. I started this project and used it to parse queries from 2015 to 2018, my current contract is frontend only and I using few SQL. I noticed this project still uses flow and I would port it to TypeScript nowadays (for sure version 1 should be implemented in TS rather than flow) but my question is: are you using it for a production project, for a real job or just as an experiment?

**Comment by papandreou** [comment]
1) I think the main use case for triple single quote is to be able to easily cram eg. a whole SQL command with both double and single quotes into a string literal:

```sql
INSERT INTO foo (bar) VALUES ('''SELECT "funkyColumnName" FROM quux WHERE thing = 'blabla'''')
```

... but I must admin that I only ran into it because my ORM uses that syntax when quoting dates.

2) I'm using it for a production project, but only for logging in development mode. The commands that get sent to the database (postgres) use the `$1`, `$2` placeholder syntax with the values sent separately. In dev I would rather see the values so the SQL log is copy/pastable for debugging, so I use this module to tokenize the query so I know where to splice in the placeholder values.

**Comment by fibo** [comment]
add branch https://github.com/fibo/SQL-tokenizer/tree/bugfix/triple-quotes

**Comment by fibo** [comment]
@papandreou sorry for the delay

just published a v1.0.0 version on npm with few improvements including support for triple quotes.

Thank again for reporting it!!
